### PR TITLE
Blog post for unit testing resource assets

### PR DESCRIPTION
**Pull request by leezen**
Resolves https://github.com/pulumi/home/issues/904

### CODE REVIEW

File: content/blog/unit-testing-assets/index.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
+---
+title: "Unit Testing Assets"
+date: 2020-06-10T11:08:26-07:00
+draft: false
+meta_desc: "Learn how unit testing your infrastructure assets can ensure help correctness of your deployments."
```
Comment (justinvp) [inline content/blog/unit-testing-assets/index.md:5]:
```suggestion
meta_desc: "Learn how unit testing your infrastructure assets can help ensure correctness of your deployments."
```

File: content/blog/unit-testing-assets/index.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+authors:
+    - lee-zen
+tags:
+    - Testing
+    - Infrastructure
+---
+
+When deploying infrastructure, we want to ensure that what we're deploying matches our expectations.
+One way to do so is via [unit testing]({{< relref "/docs/guides/testing/unit" >}}).
+We've talked about this concept in previous posts such as [in this overview]({{< relref "unit-test-infrastructure" >}})
```
Comment (justinvp) [inline content/blog/unit-testing-assets/index.md:16]: Best to fully qualify. It's unlikely we'll add `unit-test-infrastructure` elsewhere, but if we do, Hugo will get confused.

```suggestion
We've talked about this concept in previous posts such as [in this overview]({{< relref "/blog/unit-test-infrastructure" >}})
```

File: content/blog/unit-testing-assets/index.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+and [in this post on deployments with .NET]({{< relref "unit-testing-cloud-deployments-with-dotnet" >}}).
```
Comment (justinvp) [inline content/blog/unit-testing-assets/index.md:17]:
```suggestion
and [in this post on deployments with .NET]({{< relref "/blog/unit-testing-cloud-deployments-with-dotnet" >}}).
```

File: content/blog/unit-testing-assets/index.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+This helps ensure that our state never contains plaintext values that are meant to be secret. We can write a simple test
+to check this:
+
+**test_infra.py**:
+
+```python
+# Test that the secret input is marked as secret
+    @pulumi.runtime.test
+    def test_secret_input_and_output(self):
+        self.assertTrue(infra.secret_password_v1.secret_string.is_secret().result())
```
Comment (justinvp) [inline content/blog/unit-testing-assets/index.md:173]: Nit:

```suggestion
@pulumi.runtime.test
def test_secret_input_and_output(self):
    self.assertTrue(infra.secret_password_v1.secret_string.is_secret().result())
```

**Comment by leezen** [comment]
Thanks for the review and the suggestions!

**Comment by leezen** [comment]
Thanks for the suggested edits! I added an extra sentence at the very end to point to the unit testing guide.

**Pull request merged**